src/Rules/ValidYoutubeVideo.php

```diff
@@ -28,7 +28,7 @@ public function passes($attribute, $value)
     {
         try {
             $videoId = Youtube::parseVidFromURL($value);
-            $video = Youtube::getVideoInfo($videoId);
+            $video = Youtube::getVideoInfo($videoId)['id'];
```

Comment:
No, the idea was to only request the id not to assign only the id
=> Youtube::getVideoInfo($videoId, ['id']);

Reply:
Sorted this issue, thank you.